src/main/java/com/atlassian/jira/cloud/jenkins/common/client/BadRequestException.java

```diff
@@ -0,0 +1,11 @@
+package com.atlassian.jira.cloud.jenkins.common.client;
+
+public class BadRequestException extends ApiUpdateFailedException {
```

Comment:
Introduced a `BadRequestException` which is thrown when Jira returns a 400. In this case, we want to show the error message to the user.